janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/addedrelations/AddedPropertiesListValue.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Unified Catalog Team
```

Comment:
I am not sure if this is a legal requirement of your company - I presume not because in that case this shall be your company's name rather than a team's name which only makes sense within the company, but the common practice in JanusGraph is to put JanusGraph in the copyright.

Reply:
sorry, it is by mistake when I was cherry-picking from my forked version.

Reply:
Fixed